qt/scientific_interfaces/ISISReflectometry/Reduction/ReductionJobs.cpp

```diff
@@ -21,7 +23,18 @@ Group &findOrMakeGroupWithName(ReductionJobs &jobs,
     return jobs.mutableGroups()[maybeGroupIndex.get()];
   else
     return jobs.appendGroup(Group(groupName));
-} // unnamed
+}
+
+/* Return the number of rows and groups that have processing or
+ * postprocessing associated with them */
+int countItems(ReductionJobs const &jobs,
+               Item::ItemCountFunction countFunction) {
+  auto const &groups = jobs.groups();
+  return std::accumulate(groups.cbegin(), groups.cend(), 0,
+                         [countFunction](int &count, Group const &group) {
+                           return count + (group.*countFunction)();
```

Comment:
I'm not familiar with this syntax, what does this do?

Reply:
It's a function pointer to a member function in the class `Item`